library/src/main/java/com/github/pwittchen/reactivewifi/ReactiveWifi.java

```diff
@@ -42,6 +42,40 @@
 public class ReactiveWifi {
 
   /**
+   * Observe WiFi connection for enable and disable
```

Comment:
"for enable and disable is not necessary" - we can leave a comment "Observes WiFi connection". Just "s" is missing.